packages/babel-types/src/builders/index.ts

```diff
@@ -0,0 +1,5 @@
+import { numericLiteral, unaryExpression } from "./generated";
+
+export function buildUndefinedNode() {
+  return unaryExpression("void", numericLiteral(0), true);
+}
```

Comment:
Could you split this in a separate PR, that we can include in the changelog for the next Babel 7 minor, and only keep breaking changes in this one?

Reply:
Hey, do you want to prepare this for 7.23.0? :)

Reply:
Of course, I will do it right away!